apps/website-new/docs/en/configure/dts.mdx

```diff
@@ -147,6 +147,7 @@ interface DtsHostOptions {
   deleteTypesFolder?: boolean;
   maxRetries?: number;
   consumeAPITypes?: boolean;
+  remoteBasePath?: string;
```

Comment:
I looked for instructions on contributing to the documentation, but I couldn't find anything for the new website.  Hopefully this is all I need to do.  I also wasn't sure how to approach modifying the zh version.

Reply:
Chatgpt it haha thats what i do, then one of my peers who speak Chinese can alway correct any mistakes